homeassistant/components/device_tracker/aruba.py

```diff
@@ -109,20 +122,59 @@ def get_aruba_data(self):
         elif query == 6:
             _LOGGER.error("Connection timed out")
             return
-        ssh.sendline(self.password)
-        ssh.expect('#')
-        ssh.sendline('show clients')
-        ssh.expect('#')
-        devices_result = ssh.before.split(b'\r\n')
-        ssh.sendline('exit')
-
-        devices = {}
-        for device in devices_result:
-            match = _DEVICES_REGEX.search(device.decode('utf-8'))
-            if match:
-                devices[match.group('ip')] = {
-                    'ip': match.group('ip'),
-                    'mac': match.group('mac').upper(),
-                    'name': match.group('name')
-                }
+
+        if self.type == 'AP':
+            ssh.sendline(self.password)
+            ssh.expect('>')
+            ssh.sendline('enable')
+            ssh.expect('Password:')
+            ssh.sendline('enable')
+            ssh.expect('#')
+            ssh.sendline('show user')
+            ssh.sendline(' ')
+            ssh.sendline(' ')
```

Comment:
Unfortunately this is required to ensure that the controller has printed the complete output to console.